README.md

```diff
@@ -91,7 +87,7 @@ Before following these steps:
 
 ```shell
 NEW_ENV_NAME=<NAME> python samconfig.toml.d/new-dev-env.py >>samconfig.toml
-AWS_DEFAULT_REGION=eu-west-2 pipenv run sam validate
+AWS_DEFAULT_REGION=eu-west-2 uv run sam validate
```

Comment:
I have updated the README file. However, I haven't updated the docs under
https://github.com/DemocracyClub/aggregator-api/tree/master/docs
I think updating https://github.com/DemocracyClub/aggregator-api/blob/master/docs/new-development-deployment.md in particular is more than just a simple find and replace on pipenv/uv
Are these docs actually useful at this point? I must confess I have never referred to them.
If we think this is still useful, I'll try and do a pass over these to update.